### PR TITLE
Update content set URL for RC40

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

**Comment by huffman** [comment]
To test:
 1. Go through the tutorial, make sure everything looks correct
 1. When you get to the garden, you should be able to see the skybox
 1. Close interface
 1. Clear your cache
 1. Load interface, go to the garden, you should see the skybox come in fairly quickly and become sharper over time

**Comment by YuppyPlays** [comment]
Will test when ready.

**Comment by hifi-gustavo** [comment]
Build console output for commit: eb9acad42ea7ac002e51d51280dbddeb6d3698d8 available at: https://gist.github.com/bea62caf4d149058e07a0069a1be0483<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10505-eb9acad.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10505-eb9acad.dmg

**Comment by YuppyPlays** [comment]
Testing.

**Comment by YuppyPlays** [comment]
Testing complete:

When loading up the PR provided and completing the tutorial in HMD, the skybox appears completely black (you can see the 2d trees but I believe these are not a part of the skybox)

**Fail**

Can try again if required with some steps to this issue?

**Comment by birarda** [comment]
@YuppyPlays we re-posted the content set with a fix. 

Could you clear your cache (delete %APPDATA%/High Fidelity - PR10505/) and then do another test?

**Comment by YuppyPlays** [comment]
@birarda No problem at all, doing it now.

Testing.

**Comment by YuppyPlays** [comment]
Testing complete:

Crashed as soon as loading into the garden (after raising arms in the tutorial)
Tried this twice (resetting all settings inbetween) and still crashed.

Log attached in-case required.

[hifi-log.txt](https://github.com/highfidelity/hifi/files/1012983/hifi-log.txt)

**Comment by huffman** [comment]
@YuppyPlays Thank you for testing. I'm trying to reproduce what you were seeing now. Were you able to submit a crash report?

**Comment by YuppyPlays** [comment]
@huffman 

No problem at all, I did submit a crash report which stated the PR version, what happened & at the end my username I use for the game and github,

**Comment by huffman** [comment]
@YuppyPlays If you have time, could you try 2 more things?

 1. Launch again, and take note of whether it crashes
 1. Without clearing the cache or settings, start interface again and take note of whether it crashes

**Comment by marko8904** [comment]
github still occasionally triggers builds on the old cluster. This PR resulted in a duplicate. Please ignore the incoming "All checks failed" message. The real build will update the status within the next 30min. It seems that github still has some stale data on its servers.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 64bf5784e6513b56ac0a587b310ad639b03c3e96 available at: https://gist.github.com/bea62caf4d149058e07a0069a1be0483<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10505-64bf578.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10505-64bf578.dmg

**Comment by marko8904** [comment]
test this please

**Comment by huffman** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/49/HighFidelity-Beta-PR10505-64bf5784e6513b56ac0a587b310ad639b03c3e96.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/49/HighFidelity-Beta-PR10505-64bf5784e6513b56ac0a587b310ad639b03c3e96.dmg

**Comment by CainFoool** [comment]
Testing.

**Comment by RyanDowne** [comment]
This worked for me

**Comment by CainFoool** [comment]
Test Passed

Visiting the garden and letting the Skybox load, stopping the game and deleting the cache and coming back to the Garden had the Skybox load in first actually and progressively got better.

File: libraries/gpu/src/gpu/Texture_ktx.cpp [hunk 1]
```diff
@@ -255,8 +259,13 @@ void KtxStorage::assignMipData(uint16 level, const storage::StoragePointer& stor
...
 
     auto file = maybeOpenFile();
+    if (!file) {
+        qWarning() << "Failed to open file to assign mip data " << QString::fromStdString(_filename);
+        return;
+    }
 
-    auto imageData = file->mutableData();
+    auto fileData = file->mutableData();
+    auto imageData = fileData;
```
Comment (Atlante45) [inline libraries/gpu/src/gpu/Texture_ktx.cpp]: Why the extra variable?

File: libraries/gpu/src/gpu/Texture_ktx.cpp [hunk 2]
```diff
@@ -255,8 +259,13 @@ void KtxStorage::assignMipData(uint16 level, const storage::StoragePointer& stor
     }
 
     auto file = maybeOpenFile();
+    if (!file) {
+        qWarning() << "Failed to open file to assign mip data " << QString::fromStdString(_filename);
+        return;
+    }
 
-    auto imageData = file->mutableData();
+    auto fileData = file->mutableData();
```
Comment (Atlante45) [inline libraries/gpu/src/gpu/Texture_ktx.cpp]: We should probably bail if `fileData` is `nullptr`

Comment (huffman) [inline libraries/gpu/src/gpu/Texture_ktx.cpp]: Good catch

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/59/HighFidelity-Beta-PR10505-26247e86539411ec42f892390cf0e926568101ee.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/59/HighFidelity-Beta-PR10505-26247e86539411ec42f892390cf0e926568101ee.dmg

**Pull request merged**